generators/app/templates/src/services/AuthServices.ts

```diff
@@ -0,0 +1,50 @@
+import { ApiResponse } from 'apisauce';
+
+import { User, Credentials } from '~app/contexts/UserContext/reducer';
+
+import api from '../config/api';
+
+import LocalStorageService from './LocalStorageService';
+
+const TOKEN_FIELD_NAME = 'sessionToken';
+
+export interface LoginError {
+  message: string;
+}
+
+export const setCurrentUser = (currentUser: User) => {
+  api.setHeader('Authorization', currentUser.sessionToken);
+  LocalStorageService.setValue(TOKEN_FIELD_NAME, currentUser.sessionToken);
+};
+
+export const getCurrentUser = () => LocalStorageService.getValue(TOKEN_FIELD_NAME);
+
+export const removeCurrentUser = () => LocalStorageService.removeValue(TOKEN_FIELD_NAME);
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export const login = (credentials: Credentials): Promise<ApiResponse<User, LoginError>> =>
+  // TODO: Implement call to authentication API here
+  // api.post('/login', credentials );
+  new Promise(resolve => {
+    setTimeout(() => {
+      resolve({
```

Comment:
there's an existing standarized API that you can assume is being used, talk to Lucas Mujica to get the endpoints